test/unit/specs/utils/dom/selectNodesWithShadow.spec.js

```diff
@@ -136,4 +136,24 @@ describe("Utils::DOM::selectNodesWithShadow", () => {
     expect(result[0].tagName).toEqual("LABEL");
     expect(result[0].textContent).toEqual("Buy Now");
   });
+
+  it("should respect child selectors", () => {
+    const content = `
+      <div>
+        <div>
+          <span id="wrong"></span>
+        </div>
+        <span id="right"></span>
+      </div>
+    `;
+
+    const node = createNode("DIV", { id: "target" }, { innerHTML: content });
```

Comment:
I think this needs an `eq` class to allow the `afterEach` on [lines 75-77][1] to automatically clean up the markup that was added (so other tests function correctly).

Perhaps, this entire file of needs a refactor to make the purpose of the `eq` class more apparent (perhaps by renaming it to `auto-cleanup`, or change the `afterEach` to just empty the `body`, to ensure tests are executed in isolation?

[1]: https://github.com/paul-bjorkstrand/alloy/blob/main/test/unit/specs/components/Personalization/dom-actions/dom/selectNodesWithEq.spec.js#L75...L77

Reply:
@paul-bjorkstrand thanks for suggestion. Adjusted both `selectNodesWithEq.spec.js` and `selectNodesWithShadow.spec.js` to use `cleanup` CSS class to ensure `afterEach` get rids of all the elements that have been added during test run.